src/OrbitGl/App.cpp

```diff
@@ -1083,8 +1084,34 @@ void OrbitApp::StartCapture() {
 
   CHECK(capture_client_ != nullptr);
 
-  auto capture_event_processor =
-      CaptureEventProcessor::CreateForCaptureListener(this, std::move(frame_track_function_ids));
+  std::unique_ptr<CaptureEventProcessor> capture_event_processor;
+  auto capture_for_listener_processor =
```

Comment:
The order of the words is off. I suggest `processor_for_capture_listener` or something like that.